rules/windows/process_creation/proc_creation_win_apt_hurricane_panda.yml

```diff
@@ -4,25 +4,25 @@ status: test
 description: Detects Hurricane Panda Activity
 author: Florian Roth
 references:
-  - https://www.crowdstrike.com/blog/crowdstrike-discovers-use-64-bit-zero-day-privilege-escalation-exploit-cve-2014-4113-hurricane-panda/
+    - https://www.crowdstrike.com/blog/crowdstrike-discovers-use-64-bit-zero-day-privilege-escalation-exploit-cve-2014-4113-hurricane-panda/
 date: 2019/03/04
 modified: 2021/11/27
 logsource:
-  category: process_creation
-  product: windows
+    category: process_creation
+    product: windows
 detection:
-  selection:
-    - CommandLine|contains|all:
-      - 'localgroup'
-      - 'admin'
-      - '/add'
-    - CommandLine|contains:
-      - '\Win64.exe'
-  condition: selection
+    selection:
+        - CommandLine|contains|all:
+            - 'localgroup'
+            - 'admin'
+            - ''
```

Comment:
Remove the `/add` ?

Reply:
Typo it got deleted. Thanks for noticing